### ISSUE TITLE
Chrome Dev Editor. Cool New Features to Add!

### ISSUE THREAD

**Issue by Connor00116** [GoogleChrome/chromedeveditor#3988]
Version: 
OS: Chrome

Steps to reproduce:
1. You should add python!!!! I have not found anything as perfect as this program. Now I just need it in Python! I'm sure others would appreciate it too!
2. Fix that issue with the polymer not allowed by chrome or whatever it is...
3. Include a code reference, or something to look at for some codes and samples of them. Otherwise it's the best app I've seen yet!

Frequency: Always

Expected behavior: Almost there! Better than any other program! I almost expected there to be more coding languages. So please add more - or at least Python.

Actual behavior: Only Javascript and Dart... Or Polymer thingy...

Thanks, Connor00116

**Comment by richiksc** [comment]
1. This is a tool for writing apps on the web/chrome, for the web/chrome. 
2. They have also fixed that issue a long time ago with the Refactor for CSP right-click option on all projects.
3. I am not addressing this issue.

This is not a bug, why are you treating it like one? There is no expected behavior. These are all feature requests, and they are supposed to make web developer's lives easier. 

Side note:
Python is available on Chrome OS using [Python Editor](https://chrome.google.com/webstore/detail/python-editor-v4-beta/lldlpeacadpdfaoekhaiphamkndjghgo?utm_source=chrome-app-launcher-search) and the [Python NaCl](https://chrome.google.com/webstore/detail/python/nodpmmidbgeganfponihbgmfcoiibffi?utm_source=chrome-app-launcher-search), or a cloud VM such as cloud9.io and koding.com

**Comment by Connor00116** [comment]
1. I know it's a tool used for that. I am using it now more than ever and at the time I was looking for python programs. Now I looked a little more into Javascript and this app is perfect!

2. The Refractor crap doesn't work. I tried at least 10 times.

3. You aren't going to address the issue of having some kind of dictionary? I know, I know. You can go online and find everything. But what happens when you are offline? The program runs offline, but not the web. So I think you should at least have a small dictionary of codes, with the way you use them too.

Thank you for responding. I actually didn't know why I got this email. I totally forgot I commented here.
Also, I was not treating any of that stuff up there like a bug. I was only treating the Refractor as a bug.

**Comment by richiksc** [comment]
Oh sorry, I didn't address number 3 because I didn't understand what you meant. Do you mean Intellisense (autocomplete)? Remember though, this is not a tool for learning to code. I had to learn too, but after you become fluent at it, you shouldn't have to be constantly looking at a dictionary of "codes." Sure, if you're using a new library, you may need to look at the docs, but you're probably online because you had to download the library. Plus, we all run in to bugs and frustration sometimes, but you can wait until you get online and then ask the guys at StackOverflow.

About the refactor: It seems to work for me. Are you even using Polymer? When you run your app, do you get a white screen?

You were kind of treating as a bug that it doesn't support Python because it was in the 'expected behavior.'

Sorry... I was kind of critical to you last time. Thanks for replying!

Thanks,
richiksc

P.S. The plural of code is 'code' when referring to computer programming. The plural of code is 'codes' when referring to cryptography, and wartime ciphers.

**Comment by Connor00116** [comment]
Yeah, that's true. Once I became fluent (Which happened actually pretty
quickly) it wasn't that hard. I don't know about the refractor thing. I
know how to use css and html now. Thanks, man. I'll contact you if I have
anymore ideas. Also, ever made any games in whatever coding language you
use? Just asking.

**Comment by richiksc** [comment]
Yeah, I've made a game in JavaScript @ssgl. You should check it out: http://github.com/ssgl/stickdude

**Comment by Connor00116** [comment]
Good job on the game. I just checked it out. I was also just getting to
thinking that, because my dad makes websites and I help him sometimes, do
you know anyone that may need a website? I just wanted a project to do you
know. That probably sounds weird though.
